### DISCUSSION TITLE
How to Get User Message When Running Mode is Webhook?

### DISCUSSION THREAD

**Discussion by ahmadreza-fatemikia** [nutgram/nutgram#539] [Q&A]
Hello,I want to retrieve the user's message text when the running mode is webhook. The $bot->message() and $bot->update()->message do not work in webhook mode.Could you please explain how it works?

**Comment by Lukasss93** [comment]
Can you send us a complete example of the code you have written?

**Reply by ahmadreza-fatemikia** [reply]
```
$bot->stepConversation(function (Nutgram $bot) {
     $user = $bot->message()->text;
     $bot->sendMessage($user);
 });
```
But the value is null and no message is sent
